sirepo/feature_config.py

```diff
@@ -126,6 +126,7 @@ def b(msg, dev=False):
         srw=dict(
             app_url=('/en/xray-beamlines.html', str, 'URL for SRW link'),
             mask_in_toolbar=b('Show the mask element in toolbar'),
+            show_video_links=(True, bool, 'Display instruction video links'),
```

Comment:
make this false by default.

Reply:
I thought we would keep this enabled by default when BNL hosts sirepo, and only turn it off for sirepo.com

Reply:
What's the verdict?

Reply:
Off. I admin BNL, and they will need updated config when their servers are turned on again. I'll make sure it's there.

David was pretty clear he didn't want it on.